pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -646,6 +645,7 @@ func (p *PriorityQueue) movePodsToActiveOrBackoffQueue(podInfoList []*framework.
 			if err := p.activeQ.Add(pInfo); err != nil {
 				klog.ErrorS(err, "Error adding pod to the scheduling queue", "pod", klog.KObj(pod))
 			} else {
+				moved = true
```

Comment:
Could you rename it from `moved` to `activated` or `movedToActiveQ`?

Reply:
Sure, I notice there are total 3 similar wake up logics with 3 different variable names (broadcast in flushBackoffQCompleted, activated in Activate, moved in here), I'll rename them all to activated.